{"name": "test_sum_0010", "status": "skipped", "statusDetails": {"message": "XFAIL\n\nAssertionError: assert 22 == 21", "trace": "self = <test_file7.Test005 object at 0x0000015B876D0510>\n\n    @pytest.mark.regression\n    @pytest.mark.smoke\n    @pytest.mark.xfail # expected to fail\n    def test_sum_0010(self):\n        print(\"Test002\")\n        a = 12\n        b = 10\n        sum = a + b\n        print(f\"Sum of {a} and {b} is {sum}\")\n>       assert sum == 21 # fail\n        ^^^^^^^^^^^^^^^^\nE       assert 22 == 21\n\nTest_Cases\\test_file7.py:45: AssertionError"}, "attachments": [{"name": "stdout", "source": "ec93ec40-f766-490b-b61c-c26678b3800a-attachment.txt", "type": "text/plain"}], "start": 1750107917550, "stop": 1750107917551, "uuid": "a0205ced-fee0-46ed-bb9e-3ddb450381d2", "historyId": "73cd968c6ac68a4c2a26ca245c8e8641", "testCaseId": "73cd968c6ac68a4c2a26ca245c8e8641", "fullName": "Test_Cases.test_file7.Test005#test_sum_0010", "labels": [{"name": "tag", "value": "smoke"}, {"name": "tag", "value": "regression"}, {"name": "parentSuite", "value": "Test_Cases"}, {"name": "suite", "value": "test_file7"}, {"name": "subSuite", "value": "Test005"}, {"name": "host", "value": "Electricion"}, {"name": "thread", "value": "8860-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "Test_Cases.test_file7"}]}